extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -135,6 +135,15 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyala-player', 'wi
 			}
 		}
 
+		function initAttributionTracking() {
+			$('.attribution-container a').click(function () {
```

Comment:
What do you think about scoping it to `.featured-video-wrapper .attribution-container a`?
`.attribution-container` seems pretty generic to me.